indy_common/authorize/auth_map.py

```diff
@@ -202,7 +224,9 @@
     pool_config.get_action_id(): one_trustee_constraint,
     validator_info.get_action_id(): AuthConstraintOr([AuthConstraint(TRUSTEE, 1),
                                                       AuthConstraint(STEWARD, 1),
-                                                      AuthConstraint(NETWORK_MONITOR, 1)])
+                                                      AuthConstraint(NETWORK_MONITOR, 1)]),
+    create_revoc_reg_def.get_action_id(): trust_anchor_or_steward_or_trustee_owners_constraint,
```

Comment:
What does `owner` mean for RevocRegDef entry? Currently we don't require that the author of RevocRegDef is the author of corresponding CredDef (unlike the case of ReovRegEntry where we must require that the author of RevocRegEntry is also the author of the corresponding RevocRegDef).
So, maybe we should use just `trust_anchor_or_steward_or_trustee_constraint` for `create_revoc_reg_def`?